### ISSUE TITLE
Support more than 1 device

### ISSUE THREAD

**Issue by klanchman** [klanchman/homebridge-logi-circle-controls#3]
It'd be nice if the plugin could support more than 1 device.

Refer to this comment: https://github.com/klanchman/homebridge-logi-circle-controls/issues/1#issuecomment-415942579

**Comment by klanchman** [comment]
Hi @Nastras, I published a beta version of the package that supports multiple devices. If you'd like to help in testing it out, I'd greatly appreciate it. (If you can't, that's totally fine too!)

You can install the beta version by running:

```
npm i -g homebridge-logi-circle-controls@next
```

You can find the relevant documentation updates [here](https://github.com/klanchman/homebridge-logi-circle-controls/blob/develop/README.md). Essentially, you can now define `deviceId` at the accessory level, instead of only at the platform level. Example:

```jsonc
{
  "platform": "Logi Circle Controls",
  /* Email, password, etc. */
  "accessories": [{
    "type": "privacyMode",
    "name": "Privacy Mode Camera 1",
    "deviceId": "<device 1 id>"
  }, {
    "type": "privacyMode",
    "name": "Privacy Mode Camera 2",
    "deviceId": "<device 2 id>"
  }]
}
```

If you are able to test it out, please let me know here how it works.

Thanks! 😄

**Comment by Nastras** [comment]
Hello @klanchman ,
I just tested it and it works. Good job! Thank You! I get now for two cameras six devices / switches displayed when I activate all the functions in the config.
Is it possible that you create a switch for each camera Private mode and the additional features such as led, etc. are displayed with under this switch? That would save resources.

Greeting Nastra

**Comment by Nastras** [comment]
Look here, that's the fritz-platform plugin. Here, the developer has also summarized various functions under a switch.

![img_1433](https://user-images.githubusercontent.com/27686185/44952468-74e45200-ae7f-11e8-965a-5a9a7e9eafa4.jpg)

**Comment by klanchman** [comment]
Thanks for trying it out, glad it's working for you!

I went ahead and published v1.1.0, so you can simply run
```
npm i -g homebridge-logi-circle-controls
```
to get back onto the stable version.

---

I didn't realize that HomeKit supported grouping accessories under a certain device, that's nice! I think the Apple Home app doesn't actually support it, but I downloaded the Eve app and saw that the Logi Circle itself worked like that.

I looked at some other Homebridge platforms' code, and I think I might have an idea of why mine is acting differently. I'll have to spend some time looking into it.

I will create a new issue for grouping accessories under one "device". I'm not sure how quickly I'll be able to get around to it though 😅
